libcontainer/intelrdt/intelrdt.go

```diff
@@ -585,93 +613,117 @@ func (m *IntelRdtManager) Destroy() error {
 // restore the object later
 func (m *IntelRdtManager) GetPath() string {
 	if m.Path == "" {
-		m.Path, _ = GetIntelRdtPath(m.Id)
+		switch m.Type {
+		case CTRL_MON:
+			m.Path, _ = GetIntelRdtPath(m.Id)
+		case MON:
+			flattedContainerID := strings.Replace(m.Id[1:], "/", "-", -1)
+			m.Path, _ = GetMonGroupIntelRdtPath(flattedContainerID)
```

Comment:
Is it safe to omit error returned by GetMonGroupIntelRdtPath? I suggest checking this error and logging it if it not possible to return as an error from GetPath function.